### PR TITLE
This logic adds a button for the last page if the current page is the…

### LINKED ISSUE #7: The page is hiding some elements when navigating between pages
I have a table with 56 elements and configured with pageSize: 10.

My navigation bar appears as:

<< 1 2 3 4 5 >>

Seems the last page does not shows the remaining items when the total number of items is not a multiple of the page size.

Exemple:

When I click on page 1, my pageIndex = 0.

Then, when I click on page 4, my bar changes to:

<< 1 2 3 4 5 **6** >>

A new page is added.

At this point, if I click on page 6, my bar changes back to **<< 1 2 3 4 5 >>** but my items seem correct:

The problem is when I click on page 5, some items disappear from the list.

It seems that depending on the size of the item in the table, the pagination calculation does not appear to be correct, because the last page is only seen if there is 10 elements on page on this case.

---

To reproduce, create 56 dummy data and navigate between pages, see how the last item on the page disappears depending on the navigation:

image

### PR DESCRIPTION
**Pull request by erasmosoares**
… last one and ensures that the remaining items are displayed correctly.

### CODE REVIEW

File: src/TablePaginationSort/TablePaginacion.jsx [hunk 1]
```diff
@@ -198,7 +198,7 @@ const TablePaginacion = () => {
         pageIndex={pageIndex}
         setPageIndex={setPageIndex}
         totalItemsCount={data.length}
-        pageSizeOptions={[5, 25, 50]}
+        pageSizeOptions={[10, 25, 50]}
```
Comment (alepiumetti) [inline src/TablePaginationSort/TablePaginacion.jsx:201]: In this part the option is different to the initial state so, the table render 5 items and the selected option is 10 items.

**Review by alepiumetti — CHANGES_REQUESTED** [review]

**Comment by alepiumetti** [comment]
Hi! I mention the issue that this PR will resolve. 

#7 

Thnks for take your time to send a fix <3

File: src/TablePaginationSort/TablePaginacion.jsx (hunk 1 above)
Comment (erasmosoares) [reply src/TablePaginationSort/TablePaginacion.jsx:201]: You're right! It escaped on my last PR, in this case I made a new commit setting the initial state to 10!

**Comment by erasmosoares** [comment]
Fixed last change request!

Thanks 👍 :)

**Review by alepiumetti — APPROVED** [review]

**Pull request merged**